audb/core/load.py

```diff
@@ -654,6 +601,48 @@ def _tables(
     return tables
 
 
+def _update_path(
+        db: audformat.Database,
+        root: str,
+        full_path: bool,
+        format: typing.Optional[str],
+        num_workers: int,
+        verbose: bool,
+):
+    r"""Change the file path in all tables.
+
+    Args:
+        db: database object
+        root: root to add to path
+        full_path: if ``True`` expand file path with ``root``
+        format: file extension to change to in path
+        num_workers: number of workers to use
+        verbose: if ``True`` show progress bar
+
+    """
+
```

Comment:
In case `full_path` is `False` and `format` is `None` there is no need to execute the rest of the function. So maybe we should immediately return here or alternatively check it before we call this function.

Reply:
Correct, I added a check.